content/id/docs/concepts/overview/object-management-kubectl/imperative-command.md

```diff
@@ -0,0 +1,132 @@
+
+- `scale`: Melakukan *scaling* horizontal terhadap sebuah *controller* untuk menambah atau menghapus Pod dengan memperbarui jumlah replika dari *controller* tersebut.
+- `annotate`: Menambah atau menghapus anotasi sebuah objek.
+- `label`: Menambah atau menghapus label sebuah objek.
+
+Perintah `kubectl` juga mendukung perintah-perintah pembaruan berdasarkan salah satu aspek dari sebuah objek. Untuk tiap tipe objek yang berbeda, memperbarui sebuah aspek tertentu bisa berarti memperbarui sekumpulan *field* yang berbeda pula:
+
+- `set` `<field>`: Memperbarui sebuah satu aspek dari sebuah objek.
+
+{{< note >}}
+Pada Kubernetes versi 1.5, tidak semua perintah yang berdasarkan kata kerja berasosiasi dengan perintah yang berdasarkan aspek tertentu.
+{{< /note >}}
+
+*Tool* `kubectl` juga mendukung beberapa cara lain untuk memperbarui objek *live* secara langsung, meskipun cara-cara berikut membutuhkan pemahaman yang lebih tentang skema objek Kubernetes.
+
+- `edit`: Secara langsung mengedit konfigurasi mentah dari sebuah objek *live* dengan membuka konfigurasinya di sebuah editor.
+- `patch`: Secara langsung memodifikasi *field-field* spesifik dari sebuah objek *live* dengan menggunakan *patch string*. Untuk detil lebih lanjut mengenai `patch string`, lihat bagian tentang *patch* pada [Konvensi API](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#patch-operations).
+
+## Cara menghapus objek
+
+Kamu bisa menggunakan perintah `hapus` pada sebuah objek dari sebuah kluster:
+
+- `delete <type>/<name>`
+
+{{< note >}}
+Kamu bisa menggunakan `kubectl delete` baik untuk perintah imperatif maupun konfigurasi objek imperatif. Perbedaanya hanya pada argumen yang diberikan ke perintah tersebut. Untuk menggunakan `kubectl delete` sebagai perintah imperatif, argumen yang diberikan adalah objek yang hendak dihapus. Berikut adalah sebuah contoh perintah `kubectl delete` dengan sebuah objek Deployment bernama nginx sebagai argumen:
+{{< /note >}}
+
+```shell
+kubectl delete deployment/nginx
+```
+
+## Cara melihat objek
+
+{{< comment >}}
+TODO(pwittrock): Uncomment this when implemented.
+
+Kamu bisa menggunakan `kubectl view` untuk menampilkan *field-field* tertentu dari sebuah objek.
+
+- `view`: Menampilkan nilai dari *field* tertentu pada sebuah objek.
+
+{{< /comment >}}
+
+Ada beberapa perintah untuk menampilkan informasi tentang sebuah objek:
+
+- `get`: Menampilkan informasi dasar dari objek-objek yang sesuai dengan parameter dari perintah ini. Gunakan `get -h` untuk melihat daftar opsi yang bisa digunakan.
+- `describe`: Menampilkan agregat informasi detil dari objek-objek yang sesuai dengan parameter dari perintah ini.
```

Comment:
@irvifa agregat should also be standard, this is good

https://kbbi.web.id/agregat